packages/teleport/src/Console/useTabRouting.ts

```diff
@@ -7,22 +7,23 @@ You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
 
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
+    Unless required by applicable law or agreed to in writing, software
```

Comment:
Were these changes intentional or did prettier do it?

Reply:
Accidental, revised